dsc/DscForEngineers.md

```diff
@@ -1,55 +1,70 @@
 ---
-ms.date:  2017-06-12
-author:  eslesar
+ms.date:  2017-10-13
+author:  eslesar;mgreenegit
 ms.topic:  conceptual
 keywords:  dsc,powershell,configuration,setup
 title:  Desired State Configuration Overview for Decision Makers
 ---
 
-# Desired State Configuration Overview for Engineers #
+# Desired State Configuration Overview for Engineers
 
-This document is intended for developer and operations teams to understand the benefits of PowerShell Desired State Configuration (DSC).
-For a higher level view of the value DSC provides, please see [Desired State Configuration Overview for Decision Makers](decisionMaker.md)
+This document is intended for developer and operations teams
+to understand the benefits of PowerShell Desired State Configuration (DSC).
+For a higher level view of the value DSC provides,
+please see [Desired State Configuration Overview for Decision Makers](decisionMaker.md)
 
 ## Benefits of Desired State Configuration
 
 DSC exists to:
+
 - Decrease the complexity of scripting in Windows
 - Increase the speed of iteration
 
-The concept of "continuous deployment" is becoming more important. 
+The concept of "continuous deployment" is becoming more important.
 Continuous deployment means the ability to deploy frequently, potentially many times per day.
 The purpose of these deployments are not to fix something but to get something published quickly.
-By getting new features through development into operation as smoothly and reliably as possible, you reduce time-to-value of new business logic.
-
-There are two trends at play which exacerbate this need to move quickly. 
-The move towards cloud computing implies rapid change, at scale, with a constant threat of failure.
-These implications force automation in order to keep up.
-The creation of the "DevOps" mentality is a response to these changes. 
-
-
-DSC is a platform that provides declarative, autonomous and idempotent (repeatable) deployment, configuration and conformance.
-The DSC platform enables you to ensure that the components of your data center have the correct configuration, which avoids errors and prevents costly deployment failures.
-By treating DSC configurations as part of application code, DSC enables continuous deployment.
-The DSC configuration should be updated as a part of the application, ensuring that the knowledge needed to deploy the application is always up-to-date and ready to be used.
-
+By getting new features through development into operation as smoothly and reliably as possible,
+you reduce time-to-value of new business logic.
+
+The move towards cloud computing
+implies a deployment solution that utilizes a "declarative" template model,
```

Comment:
these two lines suggest a lot of complexity that doesn't get addressed. I think you can remove everything before "A declarative template model" and change the following sentences a bit so it defines declarative models and why they are good. Then talk about how that helps the cloud second? I think that leads to a better flow.